manifests/dev/vsphere-7.0u2/supervisorcluster/k8s-1.17/deploy/vsphere-csi-controller-deployment.yaml

```diff
@@ -185,6 +185,7 @@ apiVersion: v1
 data:
   "volume-extend": "true"
   "volume-health": "true"
+  "online-volume-extend": "false"
```

Comment:
Is this a mandatory field ?

Reply:
No it is not but it has been added for consistency across all versions of supervisor YAMLs

Reply:
In that case, why not add `feature=false` here `manifests/dev/vsphere-7.0u1/vanilla/deploy/vsphere-csi-controller-deployment.yaml` and so on

Reply:
All these are 7.0U2 YAMLs. We are planning to enable online resize only on 7.0U2, therefore these changes.

Reply:
My bad!